crates/supervisor/core/src/syncnode/event.rs

```diff
@@ -0,0 +1,28 @@
+use kona_interop::DerivedRefPair;
+use kona_protocol::BlockInfo;
+use kona_supervisor_types::BlockReplacement;
+
+/// Represents node events that managed nodes emits.
```

Comment:
```suggestion
/// Represents node events that [`ManagedNode`]s emits.
```
this can probably link to the helpful docs of the `ManagedNode` type thereby providing more context for whoever wants it, right, or is there some cyclical dep issue?